src/fuzgram/fuzgram.service.ts

```diff
@@ -95,4 +97,15 @@ export class FuzgramService {
     console.log(res)
     return res
   }
+
+  async socialAnalytics(username: string, social: string): Promise<void> {
```

Comment:
TODO: For this to work, we need to create this element in the document whenever a social is registered.